src/java.net.http/share/classes/java/net/http/HttpClient.java

```diff
@@ -130,9 +131,23 @@
  * privileges, should do so within an appropriate {@linkplain
  * AccessController#doPrivileged(PrivilegedAction) privileged context}.
  *
+ * @apiNote
+ * <p id="closing"> Resources allocated by the {@code HttpClient} may be
+ * reclaimed early by {@linkplain #close() closing} the client.
+ * An {@code HttpClient} instance typically manages its own pools of
+ * connections, which it can typically reuse when another request is made to
+ * the same origin server using the same protocol version and security,
+ * through the same client. In the JDK implementation, connection pools
```

Comment:
Hello Daniel, do you think we should maybe skip these details on how the connection pool decides to use a connection from pool? Perhaps, just make a mention that the HttpClient instance manages connections in a pool and may reuse it as and when necessary?

Reply:
OK